main.cpp

```diff
@@ -306,6 +307,10 @@ int main(int argc, char* argv[]) {
                 server.postPoll(fdm, nextActivity);
             }
             SINFO("Graceful bedrock shutdown complete");
+
```

Comment:
Please remove the extra line.   (See [here](https://github.com/Expensify/Style-Guide/blob/master/php.md#blank-lines))